src/main/groovy/com/ullink/gradle/sonar/SonarScannerMsbuildExtension.groovy

```diff
@@ -0,0 +1,42 @@
+/*************************************************************************
+ * ULLINK CONFIDENTIAL INFORMATION
+ * _______________________________
+ *
+ * All Rights Reserved.
+ *
+ * NOTICE: This file and its content are the property of Ullink. The
+ * information included has been classified as Confidential and may
+ * not be copied, modified, distributed, or otherwise disseminated, in
+ * whole or part, without the express written permission of Ullink.
+ ************************************************************************/
```

Comment:
Should we add the copyright on this open-source library?

Reply:
Nice catch! I removed them.